code/modules/reagents/reagent_containers/food/snacks_vr.dm

```diff
@@ -190,6 +190,19 @@
 	reagents.add_reagent("protein", 4)
 	bitesize = 2
 
+/obj/item/weapon/reagent_containers/food/snacks/grubmeat
```

Comment:
Why is this _not_ a subclass of meat?

Reply:
I'm planning on having grub meat used for 3 unique dishes. as far as I can tell if I make this a child of the meat code it'll be usable for all meat, I'm not a fan of that. xenomeat has its own seperate meat in the code